### PR TITLE
Preflight mongo version check

### PR DESCRIPTION
**Pull request by ashwathishiva**
Check mongo server version before attempting to check connectivity.

### CODE REVIEW

File: docs/preflight_checks.md [hunk 1]
```diff
@@ -110,61 +111,61 @@ Deleted pod: pod-pf-check
...
 
-### Create-RoleBinding check
+### RoleBinding check
 We use the command below to test if we are able to create a role binding in the cluster
 ```shell
-$ qliksense preflight createRoleBinding
+$ qliksense preflight rolebinding -v
 
-Preflight create roleBinding check
+Preflight oleBinding check
```
Comment (ffoysal) [inline docs/preflight_checks.md]: roleBinding

Comment (ashwathishiva) [reply docs/preflight_checks.md]: fixed typo, thank you!

File: docs/preflight_checks.md [hunk 2]
```diff
@@ -110,61 +111,61 @@ Deleted pod: pod-pf-check
...
+### Role check
 We use the command below to test if we are able to create a role in the cluster
 ```shell
-$ qliksense preflight create-role
-Preflight create-role check
+$ qliksense preflight role -v
+Preflight role check
 ---------------------------
-Preflight create-role check: 
+Preflight ole check:
```
Comment (andriy-bulynko) [inline docs/preflight_checks.md]: Same typo..

**Review by andriy-bulynko — APPROVED** [review]

File: docs/preflight_checks.md (hunk 2 above)
Comment (ashwathishiva) [reply docs/preflight_checks.md]: ah missed it! fixed that one too. Thank you!

**Pull request merged**